app/mailers/user_notifications.rb

```diff
@@ -349,13 +349,23 @@ def user_locale(user)
   end
 
   def email_post_markdown(post, add_posted_by = false)
-    result = +"#{post.raw}\n\n"
+    result = +"#{post.with_secure_media? ? strip_secure_urls(post.raw) : post.raw}\n\n"
     if add_posted_by
       result << "#{I18n.t('user_notifications.posted_by', username: post.username, post_date: post.created_at.strftime("%m/%d/%Y"))}\n\n"
     end
     result
   end
 
+  def strip_secure_urls(raw)
+    raw.each_line do |line|
+      if (line.start_with?(Discourse.store.s3_upload_host) && FileHelper.is_supported_media?(line))
```

Comment:
Are upload URLS always at the beginning of a line?

Reply:
Thanks, this should be addressed now.